mmpose/datasets/transforms/common_transforms.py

```diff
@@ -507,9 +507,9 @@ def _get_transform_params(self, bbox_scale: np.ndarray) -> Tuple:
 
         # Get scaling parameters
         scale_min, scale_max = self.scale_factor
-        scale = scale_min + (scale_max -
-                             scale_min) * self._truncnorm(size=(num_bbox, 1))
-        scale = self._truncnorm(scale_min, scale_max, size=(num_bbox, 1))
+        scale = scale_min + 0.5 * (1 + self._truncnorm(size=(num_bbox, 1))) * (
+            scale_max - scale_min)
+
```

Comment:
I think this might be better:
```Python
mu = (scale_max + scale_min) * 0.5
sigma = (scale_max - scale_min) * 0.5
scale = self._truncnorm(size=(num_bbox, 1)) * sigma + mu
```